### PR TITLE
{bio}[intel/2019b] HDDM v0.7.5 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/a655d6676dd05f0446f6c8f6a2ae1c7a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2432.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a263105733ce69d15fa4fc23df5713be for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3112.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/bf10db12f2a92f284a5ecbbe322fa309 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2432.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/50b8f7593cf1d73d8c7f2744cea52a24 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2432.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0be69d3f8a447a2c660020c0e58abb6e for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/2f9825d09703be15ec1b7799cdeaca9d for a full test report.

File: easybuild/easyconfigs/h/HDDM/HDDM-0.7.5-intel-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+easyblock = 'PythonBundle'
+
+name = 'HDDM'
+version = '0.7.5'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://ski.clps.brown.edu/hddm_docs'
+description = """HDDM is a Puthon toolbox for hierarchical Bayesian parameter estimation
```
Comment (zao) [inline easybuild/easyconfigs/h/HDDM/HDDM-0.7.5-intel-2019b-Python-3.7.4.eb]:
```suggestion
description = """HDDM is a Python toolbox for hierarchical Bayesian parameter estimation
```

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/818ae0ec2dc77e2e9f91cb0d9208b9d2 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**